demos/universal-sentence-encoder/README.md

```diff
@@ -5,12 +5,20 @@
 
 In this experiment we are going to use a TensorFlow model from [Kaggle](https://www.kaggle.com/models/google/universal-sentence-encoder/tensorFlow2/multilingual/2).
 
-```bash
+```console
 mkdir -p universal-sentence-encoder-multilingual/1/
```

Comment:
mkdir not available on windows without msys.
Use --create-dirs and chmod 755 for output on linux like in other already enabled demos.